src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/sdk/CloudSdk.java

```diff
@@ -64,9 +66,20 @@ public int runDevAppServerCommand(List<String> args) throws ProcessRunnerExcepti
     command.add(getDevAppServerPath().toString());
     command.addAll(args);
 
+    sysOutCommand(command);
+
     return processRunner.run(command.toArray(new String[command.size()]));
   }
 
+  private void sysOutCommand(List<String> command) {
+    StringBuilder commandString = new StringBuilder();
```

Comment:
Also not sure I'd want this printed out by default. Would be related to log level wouldn't it?